### DISCUSSION TITLE
How to know that I'm in a "devbox" shell ?

### DISCUSSION THREAD

**Discussion by jycouet** [JanDeDobbeleer/oh-my-posh#5559] [General]
I'm not sure what segment type I should use to get this information & style my nice (personnal taste ^^) theme.

When I'm doing `devbox shell` I'm entering this shell, (you can see as I can't start one if I'm in one already)
![image](https://github.com/user-attachments/assets/cdb6c574-65f0-45e3-9c17-ab12ea93abba)

Thank you for you help.

**Comment by JanDeDobbeleer** [comment]
@jycouet I would check if there's an environment variable set that identifies this. You can then use that inside any segment's template.

**Reply by jycouet** [reply]
I will look and keep you informed.
Thanks a lot

**Reply by jycouet** [reply]
Perfect, I added `{{ if .Env.DEVBOX_SHELL_ENABLED }} \uf487{{ end }}` and it works like a charm.

![image](https://github.com/user-attachments/assets/11af6fdf-5e50-4643-a353-bb29db15f98e)
I'm in devbox, in linux, in WSL 🤣 icons style


For info, my full config: 
https://github.com/jycouet/jycouet/blob/main/Oh_My_Posh/jyc.omp.json
